packages/notte-sdk/src/notte_sdk/endpoints/personas.py

```
@@ -169,6 +224,83 @@ def sms_read(self, persona_id: str, **data: Unpack[SMSReadRequestDict]) -> Seque
+
+    @property
+    def persona_id(self) -> str:
+        if self.info is None:
+            raise ValueError("Persona not initialized")
+        return self.info.persona_id
+
+    @override
+    def stop(self) -> None:
+        if self._init_persona_id is None:
+            logger.info(f"[Persona] {self.persona_id} deleted.")
+            _ = self.delete()
```

Pylint (3.3.7)

[error] Assigning result of a function call, where the function has no return

(E1111)